README.md

```diff
@@ -1,68 +1,64 @@
-9wm
-====
+# 9wm
 
-9wm is an X11 window manager inspired by the Plan 9 window manager 8½, also known as rio.
-It provides a very simple and clean user interface.
-It is click-to-type.
-It uses the X11 font system (which, unfortunately, means no Unicode support).
+9wm is an X11 window manager. It's inspired by the Plan 9 window manager 8½, also known as [rio].
 
-9wm does not provide virtual desktops, customization, key bindings,
-EWMH support, or compositing.
-It does not allocate any colors (if you disable COLOR),
-which will be great news if you are stuck in 1993.
+- Simple and clean user interface. It is click-to-type.
+- It uses the X11 font system (which, unfortunately, means no Unicode support).
+- No virtual desktops, customization, key bindings, EWMH support, or compositing.
+- It does not allocate any colors (if you disable COLOR), which will be great news if you are stuck in 1993.
 
-It is a great place to start if you are interested in writing a window manager from scratch:
-many folks have done exactly this.
+It is a great place to start if you are interested in writing a window manager from scratch: many folks have done exactly this.
 
-9wm is distributed under [an MIT License](LICENSE.md).
+`9wm` is distributed under [an MIT License](LICENSE.md).
 
+## Install
 
-Where do I get it?
-==================
+The latest version of 9wm is held at: <https://github.com/9wm/9wm>
 
-The latest version of 9wm is held at:
+You'll need `make`, an ANSI c compiler, `mkdir`, `cp`, `libX11` and `libXext` to build and install `9wm`. 
+In order to do so, run the following command:
 
-<https://github.com/9wm/9wm>
+```
+root # make install
+```
+You can launch `9wm` through a display manager like [lightdm] or via `startx` directly.
 
+To use `startx`, you'll need to include the following line at the end of your `.xinitrc`:
 
-How do I use it?
-================
+```sh
+exec 9wm # ... options
+```
 
-The focused window has a black border.
-Mouse and keyboard events go to this window.
-Left click a non-focused window (white border) to focus it.
+## Usage
 
-Right-click anywhere other than the focused window,
-including in any non-focused window,
-to bring up a menu.
-All further actions are done with the right mouse button.
-The menu has options for:
+You can right-click anywhere other than the focused window — including in any non-focused window — to bring up a menu. 
 
-* New: launch a new xterm
-* Reshape: select a window, draw new rectangle for its size
-* Move: drag a window around
-* Delete: kill (close) a window
-* Hide: hide (iconify) a window
+All further actions are done with the right mouse button. The menu has options for:
 
-Beneath the first 5 items are a list of all hidden windows.
+- **_New_** launchs a new terminal (default xterm)
+- **_Reshape_** resizes the focused window
+- **_Move_** drags the focused window around
+- **_Delete_** kills (close) the focused window
+- **_Hide_** hides (iconify) the focused window
```

Comment:
I would be happier with just one font change here, but I'm not convinced even that is warranted. Perhaps the patch author can present an argument for how this is a substantive improvement :)